sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_generated/models/_enums.py

```diff
@@ -107,7 +107,6 @@ class JsonWebKeyOperation(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     UNWRAP_KEY = "unwrapKey"
     IMPORT = "import"
     EXPORT = "export"
-    IMPORT_ENUM = "import"
```

Comment:
No one would use this enum value right?

Reply:
Nope! Looks like a similar case as the extra `GLOBAL_ENUM` value from Administration; this was an unused duplicate of the `IMPORT` value